app/graphql/types/submission_type.rb

```diff
@@ -31,5 +31,14 @@ class SubmissionType < Types::BaseObject
     field :user_id, String, null: false
     field :width, String, null: true
     field :year, String, null: true
+
+    field :offers, [Types::OfferType], null: false do
+      argument :gravity_partner_id, ID, required: true
+    end
+
+    def offers(gravity_partner_id:)
+      partner = Partner.find_by(gravity_partner_id: gravity_partner_id)
+      object.partner_submissions.find_by(partner: partner).offers
```

Comment:
What is `object`?

Reply:
In this case it's the `Submission` class. What you see here could be written as:

```ruby
Submission.partner_submissions.find_by(partner: partner).offers
```

But using `object` seems to be more in line with how this library wants one to use their DSL. Reading through these two docs helped me better understand how these field methods are supposed to work:

https://graphql-ruby.org/fields/introduction.html
https://graphql-ruby.org/fields/arguments.html

But let me know if I can help clarify anything further!!